### PR TITLE
chore: initial readme clean up

### PR DESCRIPTION
**Pull request by gordsport**
This PR is to update the readme (and other documentation) in the networks repo in order for it to be up-to-date or link out to content that will be kept up-to-date.

### CODE REVIEW

**Review by davidsiska-vega — APPROVED** [review]

File: contracts.md [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
-| ERC20 bridge           | Deposit assets to use them on a VEGA network                     | `0x4149257d844Ef09f11b02f2e73CbDfaB4c911a73`| [🔎](https://etherscan.io/address/0x4149257d844Ef09f11b02f2e73CbDfaB4c911a73) | [smart-contracts](https://github.com/vegaprotocol/smart-contracts) |
-| ∟ Multisig Control     | Network validators control over the bridge                       | `0x9d0707C91C67d598808834b4881348684e92E11e`| [🔎](https://etherscan.io/address/0x9d0707C91C67d598808834b4881348684e92E11e) | [smart-contracts](https://github.com/vegaprotocol/smart-contracts) |
-| ∟ Asset pool           | A dumb 'vault' that stores the assets deposited in to the bridge | `0xdee027072f42eE88bf6920b0d0C271af4e8ff8fb`| [🔎](https://etherscan.io/address/0xdee027072f42eE88bf6920b0d0C271af4e8ff8fb) | [smart-contracts](https://github.com/vegaprotocol/smart-contracts) |
+| ERC20 bridge           | Deposit assets to use them on a VEGA network                     | `0x23872549cE10B40e31D6577e0A920088B0E0666a`| [🔎](https://etherscan.io/address/0x4149257d844Ef09f11b02f2e73CbDfaB4c911a73) | [smart-contracts](https://github.com/vegaprotocol/smart-contracts) |
```
Comment (daniel1302) [inline contracts.md:6]:
```suggestion
| ERC20 bridge           | Deposit assets to use them on a VEGA network                     | `0x23872549cE10B40e31D6577e0A920088B0E0666a`| [🔎](https://etherscan.io/address/0x23872549cE10B40e31D6577e0A920088B0E0666a) | [smart-contracts](https://github.com/vegaprotocol/smart-contracts) |
```

File: contracts.md [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
+| ∟ Multisig Control     | Network validators control over the bridge                       | `0xDD2df0E7583ff2acfed5e49Df4a424129cA9B58F`| [🔎](https://etherscan.io/address/0x9d0707C91C67d598808834b4881348684e92E11e) | [smart-contracts](https://github.com/vegaprotocol/smart-contracts) |
```
Comment (daniel1302) [inline contracts.md:7]:
```suggestion
| ∟ Multisig Control     | Network validators control over the bridge                       | `0xDD2df0E7583ff2acfed5e49Df4a424129cA9B58F`| [🔎](https://etherscan.io/address/0xDD2df0E7583ff2acfed5e49Df4a424129cA9B58F) | [smart-contracts](https://github.com/vegaprotocol/smart-contracts) |
```

File: contracts.md [hunk 3]
```diff
@@ -3,17 +3,17 @@
...
+| ∟ Asset pool           | A dumb 'vault' that stores the assets deposited in to the bridge | `0xA226E2A13e07e750EfBD2E5839C5c3Be80fE7D4d`| [🔎](https://etherscan.io/address/0xdee027072f42eE88bf6920b0d0C271af4e8ff8fb) | [smart-contracts](https://github.com/vegaprotocol/smart-contracts) |
```
Comment (daniel1302) [inline contracts.md:8]:
```suggestion
| ∟ Asset pool           | A dumb 'vault' that stores the assets deposited in to the bridge | `0xA226E2A13e07e750EfBD2E5839C5c3Be80fE7D4d`| [🔎](https://etherscan.io/address/0xA226E2A13e07e750EfBD2E5839C5c3Be80fE7D4d) | [smart-contracts](https://github.com/vegaprotocol/smart-contracts) |
```

File: disaster-recovery.md [hunk 4]
```diff
@@ -39,9 +39,10 @@ In the event that the Vega master key is compromised, one can assume that the va
 3. Inform the other validators via the appropriate channels to immediately remove the compromised node from the network:
    1. Stop the network
    2. Update the genesis config file to remove the compromised node
-   3. Restore from checkpoint using the instructions [here](https://github.com/vegaprotocol/networks#restore-from-checkpoint)
-4. Use newly generated ethereum key to associate ERC20 Vega tokens from the compromised node to the new master key. 
-5. Inform all token holders that delegated to the compromised node to move their delegation. 
+   3. Non-compromised validators should restart the network from a [snapshot file](https://docs.vega.xyz/testnet/node-operators/how-to/use-snapshots#restart-a-node-using-local-snapshots)
```
Comment (daniel1302) [inline disaster-recovery.md:42]:
```suggestion
   3. Non-compromised validators should restart the network from a [snapshot file](https://docs.vega.xyz/mainnet/node-operators/how-to/use-snapshots#restart-a-node-using-local-snapshots)
```

File: disaster-recovery.md [hunk 5]
```diff
@@ -39,9 +39,10 @@ In the event that the Vega master key is compromised, one can assume that the va
...
+4. Use newly generated ethereum key to associate ERC20 Vega tokens from the compromised node to the new master key.
+5. Inform all token holders that delegated to the compromised node to move their delegation.
+6. The compromised validator should start a new node and join the network using a [snapshot file](https://docs.vega.xyz/testnet/node-operators/how-to/use-snapshots#restarting-a-node-using-a-network-snapshot) from network history.
```
Comment (daniel1302) [inline disaster-recovery.md:45]:
```suggestion
6. The compromised validator should start a new node and join the network using a [snapshot file](https://docs.vega.xyz/mainnet/node-operators/how-to/use-snapshots#restarting-a-node-using-a-network-snapshot) from network history.
```

File: disaster-recovery.md [hunk 6]
```diff
@@ -51,14 +52,14 @@ In the event a tendermint key gets compromised, a third party can double sign me
 **Note**: It is generally hard to detect a key compromise; this is normally discovered only through a general compromise (in which case all keys in working memory should be considered compromised), or by someone else using the key. In the specific case of the Tendermint key, a double signing could also stem from a misconfiguration or an error in switching between validator replicas; thus, a single double signing by a validator should be investigated, but does not require disaster recovery right away.
 
 1. Inform other validators that your tendermint key has been compromised and switch off your node.
-2. At network restart, add the new tendermint pubkey in the [genesis.json](https://github.com/vegaprotocol/networks#genesis-config-for-validator) and join others on a LNL [checkpoint restart](https://github.com/vegaprotocol/networks#restore-from-checkpoint). 
+2. At network restart, add the new tendermint pubkey in the [genesis.json](https://github.com/vegaprotocol/networks#genesis-config-for-validator) and join others using a [snapshot file](https://docs.vega.xyz/testnet/node-operators/how-to/use-snapshots#restarting-a-node-using-a-network-snapshot) from network history.
```
Comment (daniel1302) [inline disaster-recovery.md:55]:
```suggestion
2. At network restart, add the new tendermint pubkey in the [genesis.json](https://github.com/vegaprotocol/networks#genesis-config-for-validator) and join others using a [snapshot file](https://docs.vega.xyz/mainnet/node-operators/how-to/use-snapshots#restarting-a-node-using-a-network-snapshot) from network history.
```

File: disaster-recovery.md [hunk 7]
```diff
@@ -152,7 +153,7 @@ If this should occur the following actions should be taken
 2. The Vega team will replay the chain to investigate the issue
 3. The Vega team will create a new software release that fixes the bug
 4. Validators would ALL need to deploy the latest software release (assuming sucessful governance vote to deploy)
-5. Restart network using the latest [checkpoint](https://github.com/vegaprotocol/networks#restore-from-checkpoint). 
+5. Restart network using a community agreed [snapshot file](https://docs.vega.xyz/testnet/node-operators/how-to/use-snapshots#restart-a-node-using-local-snapshots).
```
Comment (daniel1302) [inline disaster-recovery.md:156]:
```suggestion
5. Restart network using a community agreed [snapshot file](https://docs.vega.xyz/mainnet/node-operators/how-to/use-snapshots#restart-a-node-using-local-snapshots).
```

File: disaster-recovery.md [hunk 8]
```diff
@@ -123,15 +124,15 @@ In the event that the genesis configuration is incorrect and this causes a major
...
+1. Restart the node(s) and see the chain data.
+2. Each node will try to replay the chain to the latest block.
 
 **NOTE:** As chain events grow, the data required to catch up will become too large for this to be a viable action.
 
 ### Scenario 3.2: If 3.1 is not viable due to age of the network
 
 1. Shut down the full network with coordination between validators.
-2. Restart the network from the last [checkpoint](https://github.com/vegaprotocol/networks#restore-from-checkpoint). 
+2. Restart the network using a community agreed [snapshot file](https://docs.vega.xyz/testnet/node-operators/how-to/use-snapshots#restart-a-node-using-local-snapshots).
```
Comment (daniel1302) [inline disaster-recovery.md:135]:
```suggestion
2. Restart the network using a community agreed [snapshot file](https://docs.vega.xyz/mainnet/node-operators/how-to/use-snapshots#restart-a-node-using-local-snapshots).
```

File: disaster-recovery.md [hunk 9]
```diff
@@ -99,22 +100,22 @@ In the event that the genesis configuration is incorrect and this causes a major
...
-### Scenario 2.4: If the incorrect configuration CANNOT be changed via governance and the parameter IS part of the checkpoint data 
+**NOTE:** This action will only work if the parameter/config is NOT part of the snapshot file data
+
+### Scenario 2.4: If the incorrect configuration CANNOT be changed via governance and the parameter IS part of the snapshot file data
 
 1. Stop the network in coordination with all validators and inform Vega team immediately
 2. Vega team to create a software release that would hardcode the value in question into the binary to resolve the issue.
 3. Validators would need to ALL deploy the latest code
-4. Restart the network using the last [checkpoint](https://github.com/vegaprotocol/networks#restore-from-checkpoint). 
+4. Restart the network using a community agreed [snapshot file](https://docs.vega.xyz/testnet/node-operators/how-to/use-snapshots#restart-a-node-using-local-snapshots).
```
Comment (daniel1302) [inline disaster-recovery.md:118]:
```suggestion
4. Restart the network using a community agreed [snapshot file](https://docs.vega.xyz/mainnet/node-operators/how-to/use-snapshots#restart-a-node-using-local-snapshots).
```

File: disaster-recovery.md [hunk 10]
```diff
@@ -99,22 +100,22 @@ In the event that the genesis configuration is incorrect and this causes a major
...
+**NOTE:** This action will only work if the parameter/config is able to be changed via a governance proposal AND time is not critical in the parameter being updated. If this is time critical it's recommended to jump to go to [Scenario 2.4.](https://github.com/vegaprotocol/networks/blob/disaster-recovery-docs/disaster-recovery.md#scenario-24-if-the-incorrec[…]-part-of-the-snapshot-file-data).
 
-### Scenario 2.3: If the incorrect configuration CANNOT be changed via a governance proposal and the parameter is NOT part of the checkpoint data
+### Scenario 2.3: If the incorrect configuration CANNOT be changed via a governance proposal and the parameter is NOT part of the snapshot file data
 
 1. Stop the network in coordination with all validators.
 2. Update the [genesis configuration](https://github.com/vegaprotocol/networks#genesis-config-for-validator)
-3. Restart the network from [checkpoint](https://github.com/vegaprotocol/networks#restore-from-checkpoint). 
+3. Restart the network from a community agreed [snapshot file](https://docs.vega.xyz/testnet/node-operators/how-to/use-snapshots#restart-a-node-using-local-snapshots).
```
Comment (daniel1302) [inline disaster-recovery.md:109]:
```suggestion
3. Restart the network from a community agreed [snapshot file](https://docs.vega.xyz/mainnet/node-operators/how-to/use-snapshots#restart-a-node-using-local-snapshots).
```

**Review by daniel1302 — COMMENTED** [review]
Looks good, a few small typos

**Pull request merged**